jbmc/src/java_bytecode/java_local_variable_table.cpp

```diff
@@ -513,18 +513,17 @@ static void populate_live_range_holes(
     merge_vars.begin(), merge_vars.end());
   std::sort(sorted_by_startpc.begin(), sorted_by_startpc.end(), lt_startpc);
 
+  PRECONDITION(!sorted_by_startpc.empty());
   maybe_add_hole(
     merge_into,
     expanded_live_range_start,
     sorted_by_startpc[0]->var.start_pc);
-  for(java_bytecode_convert_methodt::method_offsett idx = 0;
-      idx < sorted_by_startpc.size() - 1;
-      ++idx)
+  for(auto it = sorted_by_startpc.begin() + 1; it != sorted_by_startpc.end();
```

Comment:
Better `auto it = std::next(sorted_by_startpc.begin())` ?

Reply:
Thanks, done!